### ISSUE TITLE
Suggestion: add a tldr section to Coc

### ISSUE THREAD

**Issue by maneesha** [carpentries/usersguides#7]
At an admin meeting it was brought up that the Code of Conduce can be a long or intimidating read, especially if someone is feeling upset.  Suggestion is to have a 2-3 bullet point condensed version that people can easily refer to even in a stressful situation.

**Comment by maneesha** [comment]
Also CoC information is currently in three different files.  Can this be consolidated in to one file?

**Comment by ErinBecker** [comment]
@kariljordan is currently in contact with SAGE consulting to update our CoC. Any changes to the CoC should come out as a part of that process and in conversation with the Policy committee and Executive Council.

**Comment by kariljordan** [comment]
Yes, so what do you need for the handbook now from a CoC perspective? Changes are coming, but I'm sure you need something now as a placeholder that's at least the most accurate information.

**Comment by maneesha** [comment]
Thanks @kariljordan 
Nothing we need on this now.  We are tabling any changes to the CoC section pending your work with Sage.

**Comment by maneesha** [comment]
perhaps have a checklist or flow chart that lets people quickly find information, especially during a difficult situation
